### ISSUE TITLE
Static channel backups made in pending state return error when used to recover

### ISSUE THREAD

**Issue by alexbosworth** [lightningnetwork/lnd#3444]
### Background

When restoring funds using a static channel backup, in some scenarios the backup recovery method fails.

The static channel backup verification also shows this failing backup as valid

### Your environment

* lnd commit `17b6205f3a7b06b2db5f7807420629211ecb86ad`
* regtest btcd

### Steps to reproduce

1. Create a new channel between A and B
2. While the channel is still pending, grab the backup via `ExportAllChannelBackups`
3. Kill the node and wipe state
4. Attempt to restore the channel using the backup file `RestoreChannelBackups`

### Expected behavior

No error should be returned and the funds should attempt to be restored using the backup file

### Actual behavior

Restoration fails with error:

```
2 UNKNOWN: unable to unpack chan backup: a height hint greater than 0 must be provided
```

**Comment by cyclopes2011** [comment]
Not sure if this is related but I am getting a similar error after attempting to do a restore. I am currently stuck and unable to get LND running to salvage funds.

> 2019-09-19 09:56:40.301 [ERR] LTND: Unable to start server: a height hint greater than 0 must be provided
2019-09-19 09:56:40.302 [INF] RPCS: Stopping RPC Server
2019-09-19 09:56:40.310 [INF] LTND: Shutdown complete Unable to start server: a height hint greater than 0 must be provided

How do I specify a "height hint" and get back online? I have a mess of a history with a server move, return to an old server, and now a restore of a new channel.backup so I think I've messed up permanently. The new server was never successfully online, both servers were never online at the same time, etc. so I may have a slim chance of things getting back to normal.

Sorry I haven't been keeping up with the release notes so I am currently reading up on what I can find. Thank you in advance for your assistance. I can also provide more details if needed--I'm not a developer so please be specific. :)

**Comment by Roasbeef** [comment]
Fixed by #3690.
